src/sas/qtgui/Plotting/Masks/BoxMask.py

```diff
@@ -150,7 +150,7 @@ def _post_data(self):
         mask = Boxcut(x_min=x_min, x_max=x_max, y_min=y_min, y_max=y_max)
 
         if self.is_inside:
-            out = (mask(data) == False)
+            out = (not mask(data))
```

Comment:
`out` is an array of booleans. This and all other masking changes need to revert back to checking for explicit values.